app/controllers/admin/orders_controller.rb

```
@@ -0,0 +1,15 @@
+class Admin::OrdersController < ApplicationController
+
+  before_action :require_login, :require_admin
+
+  def show
+    @order = Order.find(params[:id])
+  end
+
+private
+
+  def require_admin
+    raise 'hell' unless current_user.admin?
```

probably we can just raise an error instead of hell.